src/aics-image-viewer/components/App/index.tsx

```diff
@@ -641,42 +582,6 @@ export default class App extends React.Component<AppProps, AppState> {
     }
   }
 
-  loadPrevImage(): void {
```

Comment:
hopefully this functionality can be re-implemented just by changing props from outside the component.
I believe an example of this is here:
https://imsc.allencell.org/?page=3d-viewer
(The "Stages" up/down arrows invoking next/prev image)

Reply:
I can't remember why we made it happen inside the viewer like this.  Maybe @meganrm remembers...!

Reply:
maybe it was so that the viewer internals could cache the loaded prev and next images, for faster switching?

Reply:
That looks to be correct - the `preLoad` prop seems to have controlled whether to grab surrounding images on startup, and these functions started prefetching images on switch. I guess implementing the equivalent outside the app would require exposing a `volume` prop or similar, and let the client take care of loading itself?

Reply:
Is the IMSC using website-3d-cell-viewer or base volume-viewer?

Reply:
my mistake, IMSC is directly embedding volume-viewer.  that was smart :)
 
 One initial thought I had was that website-3d-cell-viewer (or volume-viewer) could just keep a cache of Volume objects keyed by url (or LoadSpec = url + other params) and a prop could be how many Volumes to cache (`cacheSize=`).  It would evict least recently used ones when full and a new one is requested.   This would work both for time series animation playback in addition to this "next/prev" slideshow type of thing.   The key thing is that you'd have to have that single instance of the volume cache stay alive across many props changes as new Volumes are requested via website-3d-cell-viewer props.